docs/using-radicle/sharing-projects.md

```diff
@@ -0,0 +1,33 @@
+---
+id: sharing-projects
+title: Sharing projects
+---
+
+## Finding projects
+
+Projects can be shared with their Radicle ID. The Radicle ID can be copied from
+the project page.
+
+[insert picture of radicle id on project page]
+
+A Radicle ID is a URN with the following format:
+
+rad:git// etc....
+
+Copy and pasting a project's Radicle ID into your search bar will find it on the
```

Comment:
We need to make sure that we explain that in order for one to replicate a project it needs to be available. Meaning there must be a peer in your network that you can replicate it from.